spec/Appendix B -- Variable Batching.md

```diff
@@ -0,0 +1,170 @@
+## B. Appendix: Variable Batching
```

Comment:
After discussion in the composite schema working group we will have a single appendix about batching as this would allow for better example that show also both request batching and variable batching in combination.

Reply:
@benjie shall we do also some example in here with defer and stream ... I asked that because defer and stream is not yet handled in the spec.

Reply:
For now: no; but would be good to have the text handy anyway.